Makefile

```diff
@@ -428,25 +428,75 @@ validate-codeserver-image: bin/kubectl
 		fi; \
 	done ; \
 
+# R Script (test.R)
+define R_SCRIPT
+	# Set a seed for reproducibility
+	set.seed(123)
+
+	# Generate a random dataset
+	x <- rnorm(100)
+	data <- data.frame(
+	x = x,
+	y = 2 * x + rnorm(100)
+	)
+	if ("x" %in% colnames(data) && is.numeric(data$x) &&
+		"y" %in% colnames(data) && is.numeric(data$y)) {
+	# Compute basic statistics
+	mean_x <- mean(data$x, na.rm = TRUE)
+	mean_y <- mean(data$y, na.rm = TRUE)
+	correlation <- cor(data$x, data$y, use = "complete.obs")
+	# Print the statistics
+	cat("Mean of x:", mean_x, "\n")
+	cat("Mean of y:", mean_y, "\n")
+	cat("Correlation between x and y:", correlation, "\n")
+
+	} else {
+	cat("Error: Columns 'x' and 'y' must be numeric and exist in the dataset.\n")
+	}
+endef
+
+export R_SCRIPT
+
 .PHONY: validate-rstudio-image
 validate-rstudio-image: bin/kubectl
 	$(eval NOTEBOOK_NAME := $(subst .,-,$(subst cuda-,,$*)))
-	$(info # Running tests for $(NOTEBOOK_NAME) code-server image...)
+	$(info # Running tests for $(NOTEBOOK_NAME) RStudio image...)
 	$(KUBECTL_BIN) wait --for=condition=ready pod rstudio-pod --timeout=300s
 	@required_commands=$(REQUIRED_R_STUDIO_IMAGE_COMMANDS) ; \
 	if [[ $$image == "" ]] ; then \
 		echo "Usage: make validate-rstudio-image image=<container-image-name>" ; \
 		exit 1 ; \
 	fi ; \
+	echo "=> Checking container image $$image for package intallation..." ; \
+	$(KUBECTL_BIN) exec -it rstudio-pod -- mkdir -p /opt/app-root/src/R/temp-library > /dev/null 2>&1 ; \
+	$(KUBECTL_BIN) exec rstudio-pod -- R -e "install.packages('tinytex', lib='/opt/app-root/src/R/temp-library')" > /dev/null 2>&1 ; \
```

Comment:
The reason there is /opt/app-root/src/R/temp-library is that without it, the installation would fail because user does not have write access?

Reply:
Exactly, because we change the permissions on this line -> https://github.com/atheo89/notebooks/blob/main/rstudio/c9s-python-3.9/Dockerfile#L113

Reply:
So if I want to include  what I've installed in my script, I need to keep specifying this path in every include()? Doesn't R have virtualenvs? Or is this how venvs in R work?

Reply:
This temporary lib is only for the testing on the container. When I was doing manual testing on RStudio IDE itself i didn't have to create particular env for package installations.

Reply:
>  When I was doing manual testing on RStudio IDE

Why, what is different from the situation now?

Reply:
not sure